backend/src/main/java/com/huellapositiva/domain/service/EmailCommunicationService.java

```diff
@@ -92,4 +92,11 @@ public void sendConfirmationPasswordChanged(EmailAddress emailAddress) {
         EmailMessage emailMessage = EmailMessage.createFrom(from, emailAddress.toString(), UPDATE_PASS_SUBJECT, emailTemplate);
         emailService.sendEmail(emailMessage);
     }
+
+    public void sendMessageEmailChanged(EmailAddress emailAddress) {
+        final String UPDATE_EMAIL_SUBJECT = "Información sobre cambio de email";
+        EmailTemplate emailTemplate = templateService.getEmailChangedTemplate();
+        EmailMessage emailMessage = EmailMessage.createFrom(from, emailAddress.toString(), UPDATE_EMAIL_SUBJECT, emailTemplate);
+        emailService.sendEmail(emailMessage);
+    }
```

Comment:
Validation link required

Reply:
Validation link is now in the email template.